Geometry/EcalTestBeam/test/ee/SurveyToTransforms.cc

```diff
@@ -52,44 +21,29 @@
 #include "TH1.h"
 #include "TH1D.h"
 #include "TProfile.h"
-//
-// class decleration
-//
+
 using namespace CLHEP;
 
-class SurveyToTransforms : public edm::EDAnalyzer {
-  public:
-      SurveyToTransforms( const edm::ParameterSet& );
-      ~SurveyToTransforms();
+class SurveyToTransforms : public edm::one::EDAnalyzer<> {
```

Comment:
If it is using the TFileService then you must declare a 'shared resource'. For details see

https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedAnalysisEDAnalyzer#Using_TFileService